### ISSUE TITLE
tailscale ping works, but normal ping does not

### ISSUE THREAD

**Issue by tueboesen** [tailscale/tailscale#11678]
### What is the issue?

I have a remote server running Truenas scale (linux kernel), it has tailscale installed with subnet routing enabled. 

From my local machine I can ping the truenas scale server, but nothing else on the remote servers network. However, I can tailscale ping all other devices on the remote network.

When I run a ping I can see through tcpdump on the remote server that the ping is sent to the relevant machines on the remote network and that a reply is also sent back and with the correct tailscale ip address, but nothing seems to reach my local machine.

I have tested this on both a windows and ubuntu laptop on 3 different local networks, and none of them have been able to ping anything except the remote server, but can tailscale ping all devices on the remote network.

```
admin@truenas[~]$ sudo tcpdump -i br0 ip  proto \\icmp  -n 
tcpdump: verbose output suppressed, use -v[v]... for full protocol decode listening on br0, link-type EN10MB (Ethernet), snapshot length 262144 bytes 
10:47:06.453731 IP 100.68.97.36 > 192.168.0.5: ICMP echo request, id 29, seq 1, length 64 
10:47:06.453999 IP 192.168.0.5 > 100.68.97.36: ICMP echo reply, id 29, seq 1, length 64 
10:47:07.461795 IP 100.68.97.36 > 192.168.0.5: ICMP echo request, id 29, seq 2, length 64 
10:47:07.461999 IP 192.168.0.5 > 100.68.97.36: ICMP echo reply, id 29, seq 2, length 64 
10:47:08.481364 IP 100.68.97.36 > 192.168.0.5: ICMP echo request, id 29, seq 3, length 64 
10:47:08.481593 IP 192.168.0.5 > 100.68.97.36: ICMP echo reply, id 29, seq 3, length 64 
10:47:09.505723 IP 100.68.97.36 > 192.168.0.5: ICMP echo request, id 29, seq 4, length 64 
10:47:09.505888 IP 192.168.0.5 > 100.68.97.36: ICMP echo reply, id 29, seq 4, length 64 
10:47:10.529920 IP 100.68.97.36 > 192.168.0.5: ICMP echo request, id 29, seq 5, length 64 
10:47:10.530040 IP 192.168.0.5 > 100.68.97.36: ICMP echo reply, id 29, seq 5, length 64 
```

My issue seems similar to the one in https://github.com/tailscale/tailscale/issues/4181, but I don't seem to have any firewall blocking my requests. I have checked the iptables on the ubuntu machines, but I have never looked at those before and can't see anything strange there.

Both the local machines and the server are running tailscale 1.62.1.

### OS

Linux, Windows, Android

### OS version

Ubuntu 22.04, Windows 10, Truenas Scale 23.10.2

### Tailscale version

1.62.1

### Bug report
Bug report from local Ubuntu machine:
BUG-5151ea7f6def219ed7426d50fb3f1b6199c951ca828445c1ab06c724eebf9f8e-20240410062418Z-b74ae475601619e1

Bug report from Truenas scale server:
BUG-bfce094ccecd78ce6ce21e6c7c84c56d4fee844e660c694203a36c61d4bc934b-20240410062603Z-606fc87379ddb1eb

**Comment by bradfitz** [comment]
> > Bug report
>
> No response

That would help us locate your account. Ideally from both the node doing the ping and the subnet router, just after the failed ping on both sides.

There isn't enough information here to help us figure out what's going on. It might just be an ACL misconfiguration.

This looks more like a support request than a bug report at this point.

**Comment by tueboesen** [comment]
> > > Bug report
> > 
> > 
> > No response
> 
> That would help us locate your account. Ideally from both the node doing the ping and the subnet router, just after the failed ping on both sides.
> 
> There isn't enough information here to help us figure out what's going on. It might just be an ACL misconfiguration.
> 
> This looks more like a support request than a bug report at this point.

Sorry, I have edited the original now to include bug reports from both the remote server and my local ubuntu machine

**Comment by kelivel** [comment]
The bugreport on the TrueNAS server included this error:

> user bugreport health: router: could not add magicsock port rule v6: adding [-p udp --dport 48194 -j ACCEPT] in filter/ts-input: running [/sbin/ip6tables -t filter -A ts-input -p udp --dport 48194 -j ACCEPT --wait]: exit status 3: modprobe: can't change directory to '/lib/modules': No such file or directory modprobe: can't change directory to '/lib/modules': No such file or directory ip6tables v1.8.9 (legacy): can't initialize ip6tables table `filter': Table does not exist (do you need to insmod?) Perhaps ip6tables or your kernel needs to be upgraded.

This error looks similar to an issue we fixed in 1.64. If you update to the latest Tailscale version do you see the same behavior?

**Comment by dataprolet** [comment]
I'm having the same issue running Tailscale on my VPS and home server. On my VPS I can `tailscale ping` my home server's Tailscale IP and the local IP, but I can not regular ping my local IP.

I'm running the latest Docker image.

Bug report from my home server: BUG-b725769a85ddd60686e0e8b68f7f433197313afe1f128fdac862e920f652e27b-20240428112109Z-4ab4a604d081a56a
Bug report from my VPS: BUG-4e070929a885022f5cd5d4565dc93dcb39da9a2495a47b39e0426fe6a3e9df6c-20240428112343Z-d876d689f6f0c441

**Comment by tueboesen** [comment]
> The bugreport on the TrueNAS server included this error:
> 
> > user bugreport health: router: could not add magicsock port rule v6: adding [-p udp --dport 48194 -j ACCEPT] in filter/ts-input: running [/sbin/ip6tables -t filter -A ts-input -p udp --dport 48194 -j ACCEPT --wait]: exit status 3: modprobe: can't change directory to '/lib/modules': No such file or directory modprobe: can't change directory to '/lib/modules': No such file or directory ip6tables v1.8.9 (legacy): can't initialize ip6tables table `filter': Table does not exist (do you need to insmod?) Perhaps ip6tables or your kernel needs to be upgraded.
> 
> This error looks similar to an issue we fixed in 1.64. If you update to the latest Tailscale version do you see the same behavior?
Updating the version of tailscale seems to have fixed my issue, yes.

**Comment by FlorianHeigl** [comment]
today this can also be related to `TS_USERSPACE`, see:
https://www.reddit.com/r/Tailscale/comments/15f8cbc/comment/kaozlvy/
